examples/reaction_roles.py

```diff
@@ -8,27 +8,30 @@ def __init__(self, *args, **kwargs):
 
         self.role_message_id = 0  # ID of message that can be reacted to to add role
         self.emoji_to_role = {
-            partial_emoji_1: 0,  # ID of role associated with partial emoji object 'partial_emoji_1'
-            partial_emoji_2: 0  # ID of role associated with partial emoji object 'partial_emoji_2'
+            '🔴': 0,  # ID of the role associated with unicode emoji '🔴'
```

Comment:
I'm curious if there should be a mention of variation selectors and normalizing for them, most users probably don't care that there's a difference between A down arrow, and the same arrow with a variation selector applied to it. Examples here don't have this issue, but it might still be prudent to include